### PR TITLE
Added support for readPersons

### PR DESCRIPTION
**Pull request by JBlaak**

### CODE REVIEW

File: src/Requests/Imses/ReadPersonsRequest.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            throw new RequestException('No personSet in readPersons response');
+        }
+
+        $persons = [];
+        foreach ($result->personSet->person as $item) {
+            $person = new Person();
+
+            $personName = new PersonName();
+            foreach ($item->name->partName as $partName) {
+                if ($partName->namePartType === 'First') {
```
Comment (nvelthorst) [inline src/Requests/Imses/ReadPersonsRequest.php]: extract First Last And Nick and use in readPerson, readPersons and readAllPersons requests

**Comment by nvelthorst** [comment]
Travis failed

**Comment by JBlaak** [comment]
Komt doordat ik de branch heb verwijderd

**Comment by nvelthorst** [comment]
ok

**Pull request merged**